tests/test_codesigned.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (C) 2014 Anaconda, Inc
+# SPDX-License-Identifier: BSD-3-Clause
+from __future__ import annotations
+
+import os
+from functools import lru_cache
+from pathlib import Path
+from shutil import which
+from subprocess import CalledProcessError, check_output, run
+
+import pytest
+
+from conda_build.utils import on_win
+
+HERE = os.path.abspath(os.path.dirname(__file__))
```

Comment:
Should we be mixing `pathlib` and `os.path` calls in the same file? Or do we want to be consistent in usage?

Reply:
there is no Pathlib variant of shutil.which and other tests don't seem to use Pathlib much at first glance, but I'm a big proponent of Pathlib.

Reply:
We've been slowly moving paths in conda over to pathlib, but rest assured it backfired in messy code where paths are handed around and assumed strings.